sfcli/README.md

```diff
@@ -217,6 +217,16 @@ Options:
   -?|-h|--help   Show help information
 ```
 
+##### camera delete
+
+Delete a camera.
```

Comment:
also modify README around the camera base command - line 164 https://github.com/innovatrics/smartface/pull/62/files#diff-332fba503dbd59be0810ab3dc97908becbd00ac684c6181818f19b49a2e91dc9R164

Reply:
done